dev/lib/product_taxonomy/commands/generate_docs_command.rb

```diff
@@ -14,6 +14,8 @@ def initialize(options)
       super
 
       @version = validate_and_sanitize_version!(options[:version]) || UNSTABLE
+      # Versions ending with -unstable should just be "unstable" in docs
+      @version = UNSTABLE if @version.end_with?("-unstable")
```

Comment:
Rather than handling it here, let's update the callsite in `GenerateReleaseCommand` to just leave out the argument when it's calling with an unstable version.